File: Chombo/old/MoveToReleasedExamples/AMRClaw/applications/euler/2d/cylinder/mapc2p.m
function [xp,yp] = mapc2p(xc,yc)
%
% specifies the mapping to curvilinear coordinates -- should be consistent
% with mapc2p.f
%
%

theta = 2*pi*yc;
r = 16*xc + 1;

xp = r .* cos(theta);
yp = r .* sin(theta);

File: Chombo/old/MoveToReleasedExamples/AMRClaw/applications/euler/2d/cylinder/mapc2p.f
      subroutine mapc2p(xc,yc,xp,yp)
      implicit none

      double precision xc,yc,xp,yp
      double precision pi, theta, r

      common /compi/ pi

      theta = 2.d0*pi*yc
      r = 16.d0*xc + 1.d0

      xp = r * cos(theta)
      yp = r * sin(theta)

      return
      end
